### PR TITLE
doc(cli): list valid categories

### LINKED ISSUE #9209: CLI only-categories documentation
**Summary**
First time user here! While investigating how to run lighthouse from the CLI, I struggled finding what values are available to the `--only-categories` flag. `lighthouse --help` provides two: `performance,pwa`, but it's not clear what the names of the other categories are, in particular "Best Practices", because it contains a space.

Anyway, I figured it out by looking into default-config.js, but I'm interested in making this more apparent. My first thought was an update to README, but maybe an update to the `--help` output would be better since it seems like plugins can provide other categories, so it may be better to build the list dynamically, if that's even possible. Looking for input on this.

The default list, for anyone that comes across this is 
`accessibility,best-practices,performance,pwa,seo`

### PR DESCRIPTION
**Pull request by olore**
**Summary**
Small doc tweak to fix #9209 by listing the valid categories in an example

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -113,6 +113,8 @@ Examples:
   lighthouse <url> --quiet --chrome-flags="--headless"                      Launch Headless Chrome, turn off logging
   lighthouse <url> --extra-headers "{\"Cookie\":\"monster=blue\"}"          Stringify\'d JSON HTTP Header key/value pairs to send in requests
   lighthouse <url> --extra-headers=./path/to/file.json                      Path to JSON file of HTTP Header key/value pairs to send in requests
+  lighthouse <url> --only-categories=performance,pwa                        Only run specific categories. Valid categories: accessibility,
+                                                                            best-practices, performance, pwa.
```
Comment (connorjclark) [inline readme.md]:
```suggestion
                                                                            best-practices, seo, performance, pwa.
```

Comment (olore) [reply readme.md]: I didn't realize "seo" was an option! Thank you.

Is there a particular reason for your suggested rder, or is it ok if I maintain alphabetical order?

Comment (olore) [reply readme.md]: Added "seo" in alphabetical order.

File: readme.md [hunk 2]
```diff
@@ -113,6 +113,8 @@ Examples:
   lighthouse <url> --quiet --chrome-flags="--headless"                      Launch Headless Chrome, turn off logging
   lighthouse <url> --extra-headers "{\"Cookie\":\"monster=blue\"}"          Stringify\'d JSON HTTP Header key/value pairs to send in requests
   lighthouse <url> --extra-headers=./path/to/file.json                      Path to JSON file of HTTP Header key/value pairs to send in requests
+  lighthouse <url> --only-categories=performance,pwa                        Only run specific categories. Valid categories: accessibility,
```
Comment (patrickhulce) [inline readme.md]: this is periodically updated from the output of `lighthouse --help` do you think you could add the description to the actual cli-flags.js file itself? :)

Comment (olore) [reply readme.md]: Done. Sorry I should have figured that out. Thanks for your patience.

File: readme.md (hunk 1 above)
Comment (connorjclark) [reply readme.md]: alphabetical is fine. the only other acceptable order would be the order in which the categories are rendered.

File: lighthouse-cli/cli-flags.js [hunk 3]
```diff
@@ -129,7 +129,7 @@ function getFlags(manualArgv) {
         'precomputed-lantern-data-path': 'Path to the file where lantern simulation data should be read from, overwriting the lantern observed estimates for RTT and server latency.',
         'lantern-data-output-path': 'Path to the file where lantern simulation data should be written to, can be used in a future run with the `precomputed-lantern-data-path` flag.',
         'only-audits': 'Only run the specified audits',
-        'only-categories': 'Only run the specified categories',
+        'only-categories': 'Only run the specified categories. Valid categories: accessibility, best-practices, performance, pwa, seo',
```
Comment (patrickhulce) [inline lighthouse-cli/cli-flags.js]: the only tweak I might make is 
```suggestion
        'only-categories': 'Only run the specified categories. Available categories: accessibility, best-practices, performance, pwa, seo',
```

Comment (patrickhulce) [reply lighthouse-cli/cli-flags.js]: if the user has a config that adds other categories those will be valid too, so the slightly looser `Available` makes sense to me too

Comment (patrickhulce) [reply lighthouse-cli/cli-flags.js]: also, sorry the example snippet you edited in the README comes from line 61 of this file if you'd want to update that description too :)

Comment (olore) [reply lighthouse-cli/cli-flags.js]: yup, good call. I like this better

**Pull request merged**